services/rds/rds.go

```diff
@@ -502,6 +502,12 @@ func (svc *Service) updateRDSExporterConfig(tx *reform.TX, service *models.RDSSe
 	}
 	for _, n := range nodes {
 		node := n.(*models.RDSNode)
+
+		service := &models.RDSService{}
```

Comment:
```suggestion
		var service models.RDSService
```

That's a preferred syntax for zero values in PMM code base. See also https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_use_a_consistent_declaration_style